app/views/apply/eligibility/claim/light-refreshment-details.html

```diff
@@ -17,14 +17,14 @@ <h1 class="govuk-heading-l">
 
     <p>If you were away from home for more than 5 hours you may get a contribution for food and drink.</p>
 
-    <p>There’s a limit of £2.55 per person for trips over 5 hours and £5.10 for trips over 10 hours.</p>
+    <p>There is a limit of £3.50 per person for trips over 5 hours (£2.55 if visiting a Scottish prison) and £5.10 for trips over 10 hours (wherever you are travelling to).</p>
```

Comment:
Have these been changed in the code 🤔

Reply:
Not sure there are any checks - i did have a search

Reply:
ahh i didn't search in async service 😥

Reply:
I'll have a nosey 👍

Reply:
I think this one is applied manually, no reference to it that I can find